modules/javafx.controls/src/main/java/javafx/scene/control/Labeled.java

```diff
@@ -818,6 +818,48 @@ public final BooleanProperty mnemonicParsingProperty() {
         return mnemonicParsing;
     }
 
+    /**
+     * Truncated read-only property indicates whether the text has been truncated
+     * when it cannot fit into the available width.
+     * <p>
+     * When truncated, the {@link #ellipsisStringProperty() ellipsis string}
+     * gets inserted in the place dictated by the
+     * {@link #textOverrun} property.
+     *
+     * @since 23
+     * @defaultValue false
+     */
+    private ObservableBooleanValue truncated;
+
+    public final ObservableBooleanValue truncatedProperty() {
+        if (truncated == null) {
+            truncated = new BooleanBinding() {
+                {
+                    bind(
+                        ellipsisStringProperty(),
+                        fontProperty(),
+                        textProperty(),
+                        widthProperty(),
+                        wrapTextProperty()
+                    );
+                }
+
+                @Override
+                protected boolean computeValue() {
+                    if (isWrapText()) {
+                        return false;
```

Comment:
Are you sure that allowing text to wrap necessarily means it won't be truncated? What if the max height doesn't allow another line?

Reply:
Good point.  So the label will have its text truncated (by inserting the ellipsis string) when wrapText is on and the size is constrained by setting maxHeight.

Interestingly, it will not truncate the text (again, ellipsis string) if the label is resized by the layout:

![Screenshot 2024-03-05 at 12 54 14](https://github.com/openjdk/jfx/assets/107069028/1ae47317-2d54-4bcd-942d-1a49ff9faeab)

Using the latest MonkeyTester to test
https://github.com/andy-goryachev-oracle/MonkeyTest
uncomment the listener in LabelPage